CMSgov/easi-app
# [EASI-4047] Added new person role types for TRB attendees

# EASI-4047


## Changes and Description

- Added new `PersonRole` enum values in schema
- Added migration to add new values for `person_role`
- Added/updated translations
- Updated unit test



## How to test this change

- Start a TRB request
- Progress to add an attendee
- Verify options for attendee role reflect AC of ticket
- Verify data persistence
- Verify data render on label

## PR Author Review Checklist



## PR Reviewer Guidelines
- It's best to pull the branch locally and test it, rather than just looking at the code online!
- Check that all code is adequately covered by tests - if it isn't feel free to suggest the addition of tests.
- Always make comments, even if it's minor, or if you don't understand why something was done.
